components/CountdownContainer.js

```diff
@@ -60,7 +60,8 @@ function renderCountdown ({ days, hours, minutes, seconds, completed }) {
   } else {
     return (
       <div className='countdown'>
-        <span>Remaining tickets available in</span>
+        <span>Tickets sold out</span><br />
```

Comment:
Do we need this statement here?

Can we just show the countdown here and leave `Tickets sold out` to the ticket session?